examples/fluent-bit/grafana-loki/README.md

```diff
@@ -0,0 +1,23 @@
+### FireLens Example: Logging to Grafana Loki with Fluent Bit
+
+For documentation on sending your FireLens monitored log data to Grafana Loki, see: [Fluent Bit and Firelens in the Grafana documentation] (https://grafana.com/docs/loki/latest/clients/fluentbit/). The Loki data source provides access to Loki, Grafana’s log aggregation system.
```

Comment:
Link is broken because of space between `[` and `(`

Also, its always "FireLens" with 2 capitals in AWS official things

Reply:
Done.